CHANGELOG.md

```diff
@@ -1,6 +1,23 @@
 CHANGELOG
 =========
 
+# 0.31.0 / 2019-10-30
+
+* [FEATURE] Add support for SLOs [#453][] and [#464][]
```

Comment:
Really small nit, but the rest of the changelog seems to follow the order `BUGFIX` -> `FEATURE` -> `IMPROVEMENT` ordering. Can we keep that here.